target/linux/generic/hack-6.1/902-debloat_proc.patch

```diff
@@ -406,3 +406,16 @@ Signed-off-by: Felix Fietkau <nbd@nbd.name>
  	return register_pernet_subsys(&ip_rt_proc_ops);
  }
  
+--- a/net/ipv4/inet_timewait_sock.c
++++ b/net/ipv4/inet_timewait_sock.c
+@@ -272,7 +272,9 @@ static struct pernet_operations ip_rt_pr
+ 	 */
+ 
+ 	if (!rearm) {
+-		bool kill = timeo <= 4*HZ;
++#ifndef CONFIG_PROC_STRIPPED
++		bool kill = timeo <= 4*HZ;
++#endif
```

Comment:
```suggestion
@@ -272,7 +272,7 @@ static struct pernet_operations ip_rt_pr
 	 */
 
 	if (!rearm) {
-		bool kill = timeo <= 4*HZ;
+		bool  __maybe_unused kill = timeo <= 4*HZ;

```

should also do the trick with a bit less change and without if-deffery.

Reply:
Thanks! Nice idea